src/Rapidez.php

```diff
@@ -165,6 +172,16 @@ public function setStore(Store|array|callable|int|string $store): void
             }
         }
 
+        $path = config('rapidez.frontend.theme', false);
+        if ($path) {
+            config([
+                'view.paths' => [
+                    $path,
+                    ...config('view.paths'),
+                ],
+            ]);
+        }
+
```

Comment:
We'll be too late if we set the paths here
https://github.com/laravel/framework/blob/e6753fc3836bb5fdddd1ee95835d8065bd7c5b16/src/Illuminate/View/ViewServiceProvider.php#L85
https://github.com/laravel/framework/blob/e6753fc3836bb5fdddd1ee95835d8065bd7c5b16/src/Illuminate/View/FileViewFinder.php#L53

it's set and kept after the register methods. Any changes on this config after that have no effect